### DISCUSSION TITLE
Unable to run the project with ./gradlew dockerize

### DISCUSSION THREAD

**Discussion by sankeerthan2001** [eclipse-tractusx/tractusx-edc#1344] [Q&A] [answered]
FAILURE: Build failed with an exception.

* What went wrong:
A problem occurred configuring root project 'tractusx-edc'.
> Could not resolve all files for configuration ':classpath'.
   > Could not resolve org.eclipse.edc:edc-build:0.7.1-SNAPSHOT.
     Required by:
         project : > org.eclipse.edc.edc-build:org.eclipse.edc.edc-build.gradle.plugin:0.7.1-SNAPSHOT:20240527.081533-3
      > No matching variant of org.eclipse.edc:edc-build:0.7.1-SNAPSHOT:20240527.081533-6 was found. The consumer was configured to find a library for use during runtime, compatible with Java 11, packaged as a jar, and its dependencies declared externally, as well a
s attribute 'org.gradle.plugin.api-version' with value '8.5' but:
          - Variant 'apiElements' capability org.eclipse.edc:edc-build:0.7.1-SNAPSHOT declares a library, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares a component for use during compile-time, compatible with Java 17 and the consumer needed a component for use during runtime, compatible with Java 11
              - Other compatible attribute:
                  - Doesn't say anything about org.gradle.plugin.api-version (required '8.5')
          - Variant 'javadocElements' capability org.eclipse.edc:edc-build:0.7.1-SNAPSHOT declares a component for use during runtime, and its dependencies declared externally:
              - Incompatible because this component declares documentation and the consumer needed a library
              - Other compatible attributes:
                  - Doesn't say anything about its target Java version (required compatibility with Java 11)
                  - Doesn't say anything about its elements (required them packaged as a jar)
                  - Doesn't say anything about org.gradle.plugin.api-version (required '8.5')
          - Variant 'runtimeElements' capability org.eclipse.edc:edc-build:0.7.1-SNAPSHOT declares a library for use during runtime, packaged as a jar, and its dependencies declared externally:
              - Incompatible because this component declares a component, compatible with Java 17 and the consumer needed a component, compatible with Java 11
              - Other compatible attribute:
                  - Doesn't say anything about org.gradle.plugin.api-version (required '8.5')
          - Variant 'sourcesElements' capability org.eclipse.edc:edc-build:0.7.1-SNAPSHOT declares a component for use during runtime, and its dependencies declared externally:
              - Incompatible because this component declares documentation and the consumer needed a library
              - Other compatible attributes:
                  - Doesn't say anything about its target Java version (required compatibility with Java 11)
                  - Doesn't say anything about its elements (required them packaged as a jar)
                  - Doesn't say anything about org.gradle.plugin.api-version (required '8.5')

**Comment by wolf4ood** [comment, marked as answer]
Hi @sankeerthan2001 

as stated in the log message, java 17 is required to build tractusx-edc, please upgrade to that version

Thanks

**Reply by sankeerthan2001** [reply]
Hi @wolf4ood,

Thanks for the info.
* What went wrong:
Execution failed for task ':edc-controlplane:edc-controlplane-postgresql-azure-vault:dockerize'.
> Could not build image: COPY failed: file not found in build context or excluded by .dockerignore: stat build\opentelemetry-javaagent.jar: file does not exist

Getting this issue now.

**Reply by wolf4ood** [reply]
Hi @sankeerthan2001 

i would advice to use published docker images or helm charts for trying out TX-EDC and also check it out the MXD project

https://github.com/eclipse-tractusx/tutorial-resources

Thanks
